osu.Game/OsuGame.cs

```diff
@@ -485,7 +485,8 @@ protected override void LoadComplete()
                 toolbarElements.Add(d);
             });
 
-            loadComponentSingleFile(volume = new VolumeOverlay(), leftFloatingOverlayContent.Add);
+            dependencies.Cache(loadComponentSingleFile(volume = new VolumeOverlay(), leftFloatingOverlayContent.Add));
```

Comment:
```suggestion
            loadComponentSingleFile(volume = new VolumeOverlay(), leftFloatingOverlayContent.Add, true);
```

check `loadComponentSingleFile`'s implementation you can cache via a `true` parameter.